devservices/config.yml

```diff
@@ -25,8 +25,9 @@ services:
     environment:
       TASKBROKER_KAFKA_CLUSTER: "kafka-kafka-1:9093"
       TASKBROKER_CREATE_MISSING_TOPICS: "true"
+      TASKBROKER_GRPC_PORT: "50055"
```

Comment:
the only problem is getting the port to be set to 50055 for local development when running taskbroker standalone. Putting an `EXPORT TASKBROKER_GRPC_PORT=50055` in either `.envrc` or a new `.env` file did not work

I think the current approach should work for the docker container served from devservices, but I think this needs to still work for local taskbroker running side by side without requiring manual intervention from the dev

not sure the best way to accomplish that